docs/QuickStart.md

```diff
@@ -0,0 +1,58 @@
+## Getting Started / Quick Start
+
+### Features
+
+* Parses Mustache templates and an extended template format (in YAML)
+* Generates a view schema from parsed template data
+* Supports Mustache partials and sections
+* Renders templates with user-provided views
+* Validates user-provided views against generated view schema
+* Includes a [command line interface](#cli)
+
+### CLI
+
+A command line interface is provided via a `fast` binary.
+The help text is provided below and also accessed via `fast --help`:
+
+
+```
+fast <command>
+
+Commands:
+  fast validate <file>                                validate given template source file
+  fast schema <file>                                  get template parameter schema for given template source file
+  fast guiSchema <file>                               get template parameter schema (modified for use with JSON Editor) for given template source file
+  fast validateParameters <tmplFile> <parameterFile>  validate supplied template parameters with given template
+  fast render <tmplFile> [parameterFile]              render given template file with supplied parameters
+  fast validateTemplateSet <templateSetPath>          validate supplied template set
+  fast htmlpreview <tmplFile> [parameterFile]         generate a static HTML file with a preview editor to standard out
+  fast packageTemplateSet <templateSetPath> [dst]     build a package for a given template set
+
+Options:
+  --help     Show help                                                                                         [boolean]
+  --version  Show version number                                                                               [boolean]
+```
+
+For more information on a given command use the `--help` flag combined with a command:
+
+```bash
+fast <command> --help
+```
+
+The CLI can also be accessed by executing `cli.js`.
+For example:
+
+```bash
+./cli.js render path/to/template
+```
+
+### ContentTypes
+
+When BIG-IP FAST renders it is doing string replacement via Mustache, which is agnostic to the output type. However, specifying a contentType in the template can enable some additional features:
```

Comment:
@F5Debbie, should we be using "BIG-IP FAST" when referring to the SDK/language/core, or just when we are talking about the FAST product that runs on BIG-IP? I would prefer that these docs do not even mention BIG-IP to help with the messaging that `f5-fast-core` and the template language are agnostic to how the rendered output is used.

Reply:
Just for products running on BIG-IP.  No need for language etc unless we hear otherwise.

Reply:
I have changed BIG-IP FAST to FAST in subtemplating.md (3 places) and in templating.md (7 places) but there are other references to BIG-IP FAST, or just BIG-IP, that I did not remove as it references BIG-IP / BIG-IP FAST, not the FAST Core.

Reply:
Guillermo has reservations about doing this, and will discuss them with Mitchell

Reply:
I reverted the commit that changed BIG-IP FAST to FAST where referencing FAST Core.